### PR TITLE
libev: add test

### PR DESCRIPTION
**Pull request by zmwangx**
#11898.

### CODE REVIEW

File: Formula/libev.rb [hunk 1]
```diff
@@ -20,4 +20,37 @@ def install
...
+
+      int main() {
+        ev_io_init(&stdin_watcher, stdin_cb, STDIN_FILENO, EV_READ);
+        ev_io_start(EV_DEFAULT, &stdin_watcher);
+        ev_run(EV_DEFAULT, 0);
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-I#{include}", "-L#{lib}", "-lev", "-o", "test", "test.c"
+    input = "hello, world\n"
```
Comment (ilovezfs) [inline Formula/libev.rb]: same

Comment (zmwangx) [inline Formula/libev.rb]: It makes more sense to use a complete line as input.

Comment (ilovezfs) [inline Formula/libev.rb]: I'd probably do
```
output = pipe_output("./test", "hello world\n", 0)
assert_equal "hello world", output.chomp
```
but this is fine.

**Comment by ilovezfs** [comment]
:evergreen_tree:

**Pull request merged**